### PR TITLE
Expose `getRootViews and `getClientViews` on `ViewUtils`

### PR DESCRIPTION
**Pull request by teddyzeenny**

### CODE REVIEW

File: packages/ember-views/lib/index.js [hunk 1]
```diff
@@ -37,6 +39,8 @@ const ViewUtils = Ember.ViewUtils = {};
 ViewUtils.isSimpleClick = isSimpleClick;
 ViewUtils.getViewClientRects = getViewClientRects;
 ViewUtils.getViewBoundingClientRect = getViewBoundingClientRect;
+ViewUtils.getRootViews = getRootViews;
+ViewUtils.getChildViews = getChildViews;
```
Comment (chancancode) [inline packages/ember-views/lib/index.js]: @rwjblue does this seem good to you?

I think `getViewBounds` is in the same ballpark in terms of stability, so if we are adding these, maybe we should add that here too

**Comment by teddyzeenny** [comment]
Added `getViewBounds`

File: packages/ember-views/lib/index.js [hunk 2]
```diff
@@ -37,6 +40,9 @@ const ViewUtils = Ember.ViewUtils = {};
...
+ViewUtils.getViewBounds = getViewBounds;
```
Comment (chancancode) [inline packages/ember-views/lib/index.js]: nit: can you bring this above getViewClientRects? 😄

Comment (teddyzeenny) [inline packages/ember-views/lib/index.js]: done 😄

**Pull request merged**